src/react-components/ory/helpers/node.tsx

```diff
@@ -142,13 +143,20 @@ export const Node = ({
           }
         }
 
+        // the recovery code resend button
+        if (node.meta.label?.id === 1070007) {
```

Comment:
Do you think `type="reset"` on the resend button would alleviate this problem?

Reply:
Then again, I don't think we can handle that case on the kratos side, as there are fields required (CSRF, method) to be sent and "reset" doesn't send any... Maybe we can make `code´ not required?

Reply:
I think it's fine that code is required since the flow is:
1. initialize the email code recovery (required email)
2. enter code (required code input)
3 OR resend (input here isn't required).

On step 3 since we are grouping them under the same name `code` it renders it under the same form. The "best" solution i could find is this check and to specifically tell the HTML form to ignore the form validation. Another solution is to add meta information or group it under something else for the reset in Kratos, but I think thats overkill.